test/client.test.js

```diff
@@ -82,6 +82,18 @@ describe('Client', () => {
     )
   })
 
+  test('query observer', async () => {
+    var clientObserver = jest.fn()
+    var queryObserver = jest.fn()
+    var client = util.getClient({ observer: clientObserver })
+
+    await client.query(query.Now(), {
+      observer: queryObserver,
+    })
+    expect(clientObserver).toBeCalledTimes(0)
```

Comment:
Shouldn't we call both observers here? I imagine someone doing some kinda of global metrics aggregation using the `clientObserver` and a per-query metrics using `queryObserver`, as is, if we do not call the `clientObserver` the customer would need to do the global aggregation in the local too.

Reply:
on the one hand, that makes sense. but on the other hand, all options passed to query override values passed during instantiate the client object.

anyway, i commit the requested changes. let me know if it's ok, please

Reply:
@marrony what do you think?